src/main/scala/latis/service/hapi/Latis3Interpreter.scala

```diff
@@ -88,6 +96,17 @@ class Latis3Interpreter(catalog: Catalog) extends HapiInterpreter[IO] {
     enc.encode(data)
   }
 
+  override def streamJson(data: T, header: JsonObject): Stream[IO, Byte] = {
+    val dataJson = new JsonEncoder().encode(data)
+    val headerList = header.add("format", Json.fromString("json")).toList
+    val map = ListMap(headerList: _*)
+    val json2Token: Pipe[IO, Json, Token] = fs2.data.json.ast.tokenize
+    val wrapper: Pipe[IO, Token, Token] = fs2.data.json.wrap.asArrayInObject(at = "data", in = map)
+    val renderer: Pipe[IO, Token, String] = fs2.data.json.render.pretty("  ")
```

Comment:
Two spaces between the quotes "  " on purpose?

Reply:
> Two spaces between the quotes " " on purpose?

I think it had helped at one point with readability when printing out a Json to debug my code, but now it doesn't seem to be making any difference, so I'll take it out.

Reply:
https://github.com/satabin/fs2-data/blob/v1.3.1/json/shared/src/main/scala/fs2/data/json/package.scala#L156-L165

That argument is the indentation, which defaults to two spaces.

Reply:
Hmm. Idk why I originally did that then, maybe I was just trying different indentations, and settled on the default one without remembering that it was the default?